lib/chef/provisioning/docker_driver/driver.rb

```diff
@@ -70,66 +70,116 @@ def self.connection_url(driver_url)
       end
     end
 
-
     def allocate_machine(action_handler, machine_spec, machine_options)
```

Comment:
That's right . As long as this gem takes a dependency on the new version of chef-provisioning.